### PR TITLE
Login modifications on tracker to work with training application

### PR DESCRIPTION
**Pull request by Ezmeyers**
Here are the changes to tracker-capture to allow automatic login with training application, will create a pull request for training app shortly. thanks!

### CODE REVIEW

**Review by sunbiz — CHANGES_REQUESTED** [review]
I don't think we should ~~include~~ modify the gradlew, gradle-wrapper.jar, gradle-wrapper.properties and other files. ~~Can you please remove those files from the commit, or add them to .gitignore, so that they don't get committed again.~~

**Comment by Ezmeyers** [comment]
The tracker-capture hosted by DHIS2 has these included, do you still want me to remove? If so, no problem

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -116,15 +115,25 @@
...
+            android:name=".export.DBProvider"
             android:authorities="${applicationId}.export.ExportData"
             android:exported="false"
             android:grantUriPermissions="true">
             <meta-data
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_paths" />
         </provider>
+
+        <receiver
```
Comment (sunbiz) [inline app/src/main/AndroidManifest.xml:129]: Only this change is required. All the other changes are not appropriate

File: app/src/main/java/org/hisp/dhis/android/trackercapture/MainActivity.java [hunk 2]
```diff
@@ -35,20 +35,26 @@
 import android.content.Intent;
```
Comment (sunbiz) [inline app/src/main/java/org/hisp/dhis/android/trackercapture/MainActivity.java:35]: No change is required to MainActivity.java

File: app/gradlew [hunk 3]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (sunbiz) [inline app/gradlew:1]: No changes required to app/gradlew

File: app/gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#Thu Nov 30 01:08:05 EST 2017
```
Comment (sunbiz) [inline app/gradle/wrapper/gradle-wrapper.properties:1]: No changes required to app/gradle/wrapper/gradle-wrapper.properties

File: app/gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (sunbiz) [inline app/gradlew.bat:1]: No changes are required to app/gradlew.bat

File: app/proguard-rules.pro [hunk 6]
```diff
@@ -15,3 +15,5 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
```
Comment (sunbiz) [inline app/proguard-rules.pro:15]: No changes required to app/proguard-rules.pro

**Review by sunbiz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**